### ISSUE TITLE
Enhance a11y for `useWindowEvent`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5402]
name: useWindowEvent

### Description
1 accessibility violation was found in the `useWindowEvent`.

#### [Hooks / useWindowEvent / Basic](https://yamada-ui.github.io/yamada-ui/?path=/story/hooks-usewindowevent--basic)

Received:
Form elements must have labels (label)

Fix any of the following:
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/label?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
